kernel: bump 6.6 to 6.6.36

graysky2 opened the pull request:
Changelog: https://cdn.kernel.org/pub/linux/kernel/v6.x/ChangeLog-6.6.36

Manually rebased:
	generic/hack-6.6/765-mxl-gpy-control-LED-reg-from-DT.patch
	bcm27xx/patches-6.6/950-0536-dmaengine-dw-axi-dmac-Fixes-for-RP1.patch

Removed upstreamed:
	bmips/patches-6.6/010-v6.10-mips-bmips-BCM6358-make-sure-CBR-is-correctly-set.patch[1]

All other patches automatically rebased.

1. https://github.com/openwrt/openwrt/commit/7c9644a7b5e557f05b818296287cdebdd38f1097

Build system: x86/64
Build-tested: x86/64/AMD Cezanne, flogic/xiaomi_redmi-router-ax6000-ubootmod, ramips/tplink_archer-a6-v3
Run-tested: x86/64/AMD Cezanne, flogic/xiaomi_redmi-router-ax6000-ubootmod, ramips/tplink_archer-a6-v3

graysky2 commented:
I do not believe my rebase of `generic/hack-6.6/765-mxl-gpy-control-LED-reg-from-DT.patch` is correct as I cannot build `flogic/xiaomi_redmi-router-ax6000-ubootmod`:
```
...
make -C /scratch/union/build_dir/target-aarch64_cortex-a53_musl/linux-mediatek_filogic/linux-6.6.36 KCFLAGS="-fmacro-prefix-map=/scratch/union/build_dir/target-aarch64_cortex-a53_musl=target-aarch64_cortex-a53_musl -mcpu=cortex-a53 -mcpu=cortex-a53" HOSTCFLAGS="-O2 -I/scratch/union/staging_dir/host/include  -Wall -Wmissing-prototypes -Wstrict-prototypes" CROSS_COMPILE="aarch64-openwrt-linux-musl-" ARCH="arm64" KBUILD_HAVE_NLS=no KBUILD_BUILD_USER="" KBUILD_BUILD_HOST="" KBUILD_BUILD_TIMESTAMP="Thu Jun 27 14:11:11 2024" KBUILD_BUILD_VERSION="0" KBUILD_HOSTLDFLAGS="-L/scratch/union/staging_dir/host/lib" CONFIG_SHELL="bash" V=''  cmd_syscalls=  CC="aarch64-openwrt-linux-musl-gcc" KERNELRELEASE=6.6.36 Image dtbs modules
make[5]: Entering directory '/scratch/union/build_dir/target-aarch64_cortex-a53_musl/linux-mediatek_filogic/linux-6.6.36'
  CC      drivers/net/phy/mxl-gpy.o
drivers/net/phy/mxl-gpy.c: In function 'gpy_ack_interrupt':
drivers/net/phy/mxl-gpy.c:241:9: error: unknown type name 'eturn'
  241 |         eturn gpy_led_write(phydev);
      |         ^~~~~
drivers/net/phy/mxl-gpy.c:241:9: error: parameter names (without types) in function declaration [-Werror]
drivers/net/phy/mxl-gpy.c: At top level:
drivers/net/phy/mxl-gpy.c:283:12: error: static declaration of 'gpy_led_write' follows non-static declaration
  283 | static int gpy_led_write(struct phy_device *phydev)
      |            ^~~~~~~~~~~~~
drivers/net/phy/mxl-gpy.c:241:15: note: previous declaration of 'gpy_led_write' with type 'int()'
  241 |         eturn gpy_led_write(phydev);
      |               ^~~~~~~~~~~~~
drivers/net/phy/mxl-gpy.c: In function 'gpy_ack_interrupt':
drivers/net/phy/mxl-gpy.c:242:1: error: control reaches end of non-void function [-Werror=return-type]
  242 | }
      | ^
drivers/net/phy/mxl-gpy.c: At top level:
drivers/net/phy/mxl-gpy.c:283:12: error: 'gpy_led_write' defined but not used [-Werror=unused-function]
  283 | static int gpy_led_write(struct phy_device *phydev)
      |            ^~~~~~~~~~~~~
cc1: all warnings being treated as errors
make[10]: *** [scripts/Makefile.build:243: drivers/net/phy/mxl-gpy.o] Error 1
make[9]: *** [scripts/Makefile.build:480: drivers/net/phy] Error 2
make[8]: *** [scripts/Makefile.build:480: drivers/net] Error 2
make[7]: *** [scripts/Makefile.build:480: drivers] Error 2
make[6]: *** [/scratch/union/build_dir/target-aarch64_cortex-a53_musl/linux-mediatek_filogic/linux-6.6.36/Makefile:1921: .] Error 2
make[5]: *** [Makefile:234: __sub-make] Error 2
make[5]: Leaving directory '/scratch/union/build_dir/target-aarch64_cortex-a53_musl/linux-mediatek_filogic/linux-6.6.36'
make[4]: *** [Makefile:18: /scratch/union/build_dir/target-aarch64_cortex-a53_musl/linux-mediatek_filogic/linux-6.6.36/.modules] Error 2
make[4]: Leaving directory '/scratch/union/target/linux/mediatek'
make[3]: *** [Makefile:11: compile] Error 2
make[3]: Leaving directory '/scratch/union/target/linux'
time: target/linux/compile#3.59#2.52#6.46
    ERROR: target/linux failed to build.
make[2]: *** [target/Makefile:32: target/linux/compile] Error 1
make[2]: Leaving directory '/scratch/union'
make[1]: *** [target/Makefile:25: /scratch/union/staging_dir/target-aarch64_cortex-a53_musl/stamp/.target_compile] Error 2
make[1]: Leaving directory '/scratch/union'
make: *** [/scratch/union/include/toplevel.mk:248: world] Error 2
```

dannil commented on target/linux/generic/hack-6.6/765-mxl-gpy-control-LED-reg-from-DT.patch:
```diff
@@ -55,7 +55,19 @@ Signed-off-by: David Bauer <mail@david-bauer.net>
++	if (ret < 0)
++		return ret;
++
++	eturn gpy_led_write(phydev);
```
```suggestion
+	return gpy_led_write(phydev);
```

graysky2 replied:
I made the proposed modification but building still fails.
```
...
make -C /scratch/union/build_dir/target-aarch64_cortex-a53_musl/linux-mediatek_filogic/linux-6.6.36 KCFLAGS="-fmacro-prefix-map=/scratch/union/build_dir/target-aarch64_cortex-a53_musl=target-aarch64_cortex-a53_musl -mcpu=cortex-a53 -mcpu=cortex-a53" HOSTCFLAGS="-O2 -I/scratch/union/staging_dir/host/include  -Wall -Wmissing-prototypes -Wstrict-prototypes" CROSS_COMPILE="aarch64-openwrt-linux-musl-" ARCH="arm64" KBUILD_HAVE_NLS=no KBUILD_BUILD_USER="" KBUILD_BUILD_HOST="" KBUILD_BUILD_TIMESTAMP="Thu Jun 27 14:11:11 2024" KBUILD_BUILD_VERSION="0" KBUILD_HOSTLDFLAGS="-L/scratch/union/staging_dir/host/lib" CONFIG_SHELL="bash" V=''  cmd_syscalls=  CC="aarch64-openwrt-linux-musl-gcc" KERNELRELEASE=6.6.36 Image dtbs modules
make[5]: Entering directory '/scratch/union/build_dir/target-aarch64_cortex-a53_musl/linux-mediatek_filogic/linux-6.6.36'
  CC      drivers/net/phy/mxl-gpy.o
drivers/net/phy/mxl-gpy.c: In function 'gpy_ack_interrupt':
drivers/net/phy/mxl-gpy.c:241:16: error: implicit declaration of function 'gpy_led_write' [-Werror=implicit-function-declaration]
  241 |         return gpy_led_write(phydev);
      |                ^~~~~~~~~~~~~
drivers/net/phy/mxl-gpy.c: At top level:
drivers/net/phy/mxl-gpy.c:283:12: error: static declaration of 'gpy_led_write' follows non-static declaration
  283 | static int gpy_led_write(struct phy_device *phydev)
      |            ^~~~~~~~~~~~~
drivers/net/phy/mxl-gpy.c:241:16: note: previous implicit declaration of 'gpy_led_write' with type 'int()'
  241 |         return gpy_led_write(phydev);
      |                ^~~~~~~~~~~~~
drivers/net/phy/mxl-gpy.c:283:12: error: 'gpy_led_write' defined but not used [-Werror=unused-function]
  283 | static int gpy_led_write(struct phy_device *phydev)
      |            ^~~~~~~~~~~~~
cc1: all warnings being treated as errors
make[10]: *** [scripts/Makefile.build:243: drivers/net/phy/mxl-gpy.o] Error 1
make[9]: *** [scripts/Makefile.build:480: drivers/net/phy] Error 2
make[8]: *** [scripts/Makefile.build:480: drivers/net] Error 2
make[7]: *** [scripts/Makefile.build:480: drivers] Error 2
make[6]: *** [/scratch/union/build_dir/target-aarch64_cortex-a53_musl/linux-mediatek_filogic/linux-6.6.36/Makefile:1921: .] Error 2
make[5]: *** [Makefile:234: __sub-make] Error 2
make[5]: Leaving directory '/scratch/union/build_dir/target-aarch64_cortex-a53_musl/linux-mediatek_filogic/linux-6.6.36'
make[4]: *** [Makefile:18: /scratch/union/build_dir/target-aarch64_cortex-a53_musl/linux-mediatek_filogic/linux-6.6.36/.modules] Error 2
make[4]: Leaving directory '/scratch/union/target/linux/mediatek'
make[3]: *** [Makefile:11: compile] Error 2
make[3]: Leaving directory '/scratch/union/target/linux'
time: target/linux/compile#3.51#2.40#6.24
    ERROR: target/linux failed to build.
make[2]: *** [target/Makefile:32: target/linux/compile] Error 1
make[2]: Leaving directory '/scratch/union'
make[1]: *** [target/Makefile:25: /scratch/union/staging_dir/target-aarch64_cortex-a53_musl/stamp/.target_compile] Error 2
make[1]: Leaving directory '/scratch/union'
make: *** [/scratch/union/include/toplevel.mk:248: world] Error 2
```

zxlhhyccc replied:
66 -	return gpy_led_write(phydev);

zxlhhyccc commented on target/linux/generic/hack-6.6/765-mxl-gpy-control-LED-reg-from-DT.patch:
```diff
@@ -90,16 +102,4 @@ Signed-off-by: David Bauer <mail@david-bauer.net>
- }
- 
- static int gpy_probe(struct phy_device *phydev)
+ 	/* Nothing to configure. Configuration Requirement Placeholder */
```
106 +	return gpy_led_write(phydev);

graysky2 replied:
I inserted this line and corrected the one below but I am still getting a build failure:
```
...
make -C /scratch/union/build_dir/target-aarch64_cortex-a53_musl/linux-mediatek_filogic/linux-6.6.36 KCFLAGS="-fmacro-prefix-map=/scratch/union/build_dir/target-aarch64_cortex-a53_musl=target-aarch64_cortex-a53_musl -mcpu=cortex-a53 -mcpu=cortex-a53" HOSTCFLAGS="-O2 -I/scratch/union/staging_dir/host/include  -Wall -Wmissing-prototypes -Wstrict-prototypes" CROSS_COMPILE="aarch64-openwrt-linux-musl-" ARCH="arm64" KBUILD_HAVE_NLS=no KBUILD_BUILD_USER="" KBUILD_BUILD_HOST="" KBUILD_BUILD_TIMESTAMP="Thu Jun 27 14:11:11 2024" KBUILD_BUILD_VERSION="0" KBUILD_HOSTLDFLAGS="-L/scratch/union/staging_dir/host/lib" CONFIG_SHELL="bash" V=''  cmd_syscalls=  CC="aarch64-openwrt-linux-musl-gcc" KERNELRELEASE=6.6.36 Image dtbs modules
make[5]: Entering directory '/scratch/union/build_dir/target-aarch64_cortex-a53_musl/linux-mediatek_filogic/linux-6.6.36'
  CC      drivers/net/phy/mxl-gpy.o
drivers/net/phy/mxl-gpy.c: In function 'gpy_ack_interrupt':
drivers/net/phy/mxl-gpy.c:241:16: error: implicit declaration of function 'gpy_led_write' [-Werror=implicit-function-declaration]
  241 |         return gpy_led_write(phydev);
      |                ^~~~~~~~~~~~~
drivers/net/phy/mxl-gpy.c: At top level:
drivers/net/phy/mxl-gpy.c:283:12: error: static declaration of 'gpy_led_write' follows non-static declaration
  283 | static int gpy_led_write(struct phy_device *phydev)
      |            ^~~~~~~~~~~~~
drivers/net/phy/mxl-gpy.c:241:16: note: previous implicit declaration of 'gpy_led_write' with type 'int()'
  241 |         return gpy_led_write(phydev);
      |                ^~~~~~~~~~~~~
cc1: all warnings being treated as errors
make[10]: *** [scripts/Makefile.build:243: drivers/net/phy/mxl-gpy.o] Error 1
make[9]: *** [scripts/Makefile.build:480: drivers/net/phy] Error 2
make[8]: *** [scripts/Makefile.build:480: drivers/net] Error 2
make[7]: *** [scripts/Makefile.build:480: drivers] Error 2
make[6]: *** [/scratch/union/build_dir/target-aarch64_cortex-a53_musl/linux-mediatek_filogic/linux-6.6.36/Makefile:1921: .] Error 2
make[5]: *** [Makefile:234: __sub-make] Error 2
make[5]: Leaving directory '/scratch/union/build_dir/target-aarch64_cortex-a53_musl/linux-mediatek_filogic/linux-6.6.36'
make[4]: *** [Makefile:18: /scratch/union/build_dir/target-aarch64_cortex-a53_musl/linux-mediatek_filogic/linux-6.6.36/.modules] Error 2
make[4]: Leaving directory '/scratch/union/target/linux/mediatek'
make[3]: *** [Makefile:11: compile] Error 2
make[3]: Leaving directory '/scratch/union/target/linux'
time: target/linux/compile#3.47#2.51#6.32
    ERROR: target/linux failed to build.
make[2]: *** [target/Makefile:32: target/linux/compile] Error 1
make[2]: Leaving directory '/scratch/union'
make[1]: *** [target/Makefile:25: /scratch/union/staging_dir/target-aarch64_cortex-a53_musl/stamp/.target_compile] Error 2
make[1]: Leaving directory '/scratch/union'
make: *** [/scratch/union/include/toplevel.mk:248: world] Error 2
```

zxlhhyccc replied:
This patch uncompiled `gpy_led_write`, so add `return gpy_led_write(phydev); `To `static int gpy_config_init(struct phy_device *phydev)`,

see: https://github.com/openwrt/openwrt/blob/dd1a56c550f2ec34ce5d76e1c763806ab8e29ae4/target/linux/generic/hack-6.1/765-mxl-gpy-control-LED-reg-from-DT.patch

Noltari commented:
> bmips/patches-6.6/203-mips-bmips-dma-fix-CBR-address.patch

@graysky2 this patch has been removed in 7c9644a7b5e557f05b818296287cdebdd38f1097

zxlhhyccc commented on target/linux/generic/hack-6.6/765-mxl-gpy-control-LED-reg-from-DT.patch:
```diff
@@ -63,11 +63,11 @@ Signed-off-by: David Bauer <mail@david-bauer.net>
 +		return ret;
 +
-+	eturn gpy_led_write(phydev);
++	return gpy_led_write(phydev);
```
Following the error message, this line should be deleted.
66 - +  return gpy_led_write(phydev);

graysky2 replied:
OK ... should the new line inserted under ` 	/* Nothing to configure. Configuration Requirement Placeholder */` remain?

zxlhhyccc replied:
> OK ... should the new line inserted under ` /* Nothing to configure. Configuration Requirement Placeholder */` remain?

Latest patch this line deleted:
https://github.com/openwrt/openwrt/pull/15818/files/19031bd08571aa8e7868d71d5e8d4ce45371d579..64ef2eb5080f580a048163942addcd8bd376c601#diff-8c7f9f802cfa866c19fb895d0477a1f743db82a320c06fa6c68bea69d1e78080R66

graysky2 committed:
kernel: bump 6.6 to 6.6.36

Changelog: https://cdn.kernel.org/pub/linux/kernel/v6.x/ChangeLog-6.6.36

Manually rebased:
        generic/hack-6.6/765-mxl-gpy-control-LED-reg-from-DT.patch
        bcm27xx/patches-6.6/950-0536-dmaengine-dw-axi-dmac-Fixes-for-RP1.patch

Removed upstreamed:
	bmips/patches-6.6/010-v6.10-mips-bmips-BCM6358-make-sure-CBR-is-correctly-set.patch[1]

All other patches automatically rebased.

1. https://github.com/openwrt/openwrt/commit/7c9644a7b5e557f05b818296287cdebdd38f1097

Build system: x86/64
Build-tested: x86/64/AMD Cezanne, flogic/xiaomi_redmi-router-ax6000-ubootmod, ramips/tplink_archer-a6-v3
Run-tested: x86/64/AMD Cezanne, flogic/xiaomi_redmi-router-ax6000-ubootmod, ramips/tplink_archer-a6-v3

Signed-off-by: John Audia <therealgraysky@proton.me>

blocktrron commented on target/linux/generic/hack-6.6/765-mxl-gpy-control-LED-reg-from-DT.patch:
```diff
@@ -55,7 +55,17 @@ Signed-off-by: David Bauer <mail@david-bauer.net>
  #define PHY_MDI_MDI_X_MASK	GENMASK(1, 0)
  #define PHY_MDI_MDI_X_NORMAL	0x3
-@@ -260,6 +267,35 @@ out:
+@@ -228,7 +235,8 @@ static int gpy_ack_interrupt(struct phy_
+ 
+ 	/* Clear all pending interrupts */
+ 	ret = phy_read(phydev, PHY_ISTAT);
+-	return ret < 0 ? ret : 0;
++	if (ret < 0)
++		return ret;
+ }
+ 
+ static int gpy_mbox_read(struct phy_device *phydev, u32 addr)
+@@ -270,6 +278,35 @@ out:
```
THis diff does not belong here, this function now misses a return value.

blocktrron commented on target/linux/generic/hack-6.6/765-mxl-gpy-control-LED-reg-from-DT.patch:
```diff
@@ -90,16 +100,4 @@ Signed-off-by: David Bauer <mail@david-bauer.net>
- 
- 	/* Clear all pending interrupts */
- 	ret = phy_read(phydev, PHY_ISTAT);
--	return ret < 0 ? ret : 0;
-+	if (ret < 0)
-+		return ret;
-+
-+	return gpy_led_write(phydev);
```
This still belongs in gpy_config_init

zxlhhyccc replied:
@graysky2 
Change to the following format:
````
@@ -216,7 +223,10 @@ static int gpy_ack_interrupt(struct phy_

 	/* Clear all pending interrupts */
 	ret = phy_read(phydev, PHY_ISTAT);
-	return ret < 0 ? ret : 0;
+	if (ret < 0)
+		return ret;
 }
````
。。。。。。
````
 static int gpy_config_init(struct phy_device *phydev)
 {
 -	/* Nothing to configure. Configuration Requirement Placeholder */
 - 		return 0
 +    return gpy_led_write(phydev);
}

namiltd commented:
- Refresh: target/linux/mediatek/patches-6.6/732-net-phy-mxl-gpy-don-t-use-SGMII-AN-if-using-phylink.patch
 - Add "#sound-dai-cells = <0>; "below line "wm8960: wm8960@1a {"
to 863-arm64-dts-mt7986-add-sound-wm8960.patch and 864-arm64-dts-mt7986-add-sound-overlay-for-bpi-r3.patch

DragonBluep commented:
[0001-fix.patch](https://github.com/user-attachments/files/16038356/0001-fix.patch) Please squash this patch.

zxlhhyccc commented:
> [0001-fix.patch](https://github.com/user-attachments/files/16038356/0001-fix.patch)请修复此补丁。

This patch should be wrong.

zxlhhyccc commented:
@graysky2 The current 765 patch uncompiled `gpy_led_write`, so add `return gpy_led_write(phydev); `To `static int gpy_config_init(struct phy_device *phydev),`

see: https://github.com/openwrt/openwrt/blob/dd1a56c550f2ec34ce5d76e1c763806ab8e29ae4/target/linux/generic/hack-6.1/765-mxl-gpy-control-LED-reg-from-DT.patch

zxlhhyccc commented:
> I successfully tested the operation of kernel 6.6.36 on ramips, mt7620, TP-Link Archer C5 v4

The current patch does not affect the device compilation issues you mentioned.

zxlhhyccc commented:
@graysky2 I modified the patch, seems to solve the problem, the patch is as follows:
[765-mxl-gpy-control-LED-reg-from-DT.patch](https://github.com/user-attachments/files/16038957/765-mxl-gpy-control-LED-reg-from-DT.patch)

graysky2 commented:
@DragonBluep -
> [0001-fix.patch](https://github.com/user-attachments/files/16038356/0001-fix.patch) Please squash this patch.
But it fails to compile for me:
```
Applying /scratch/union/target/linux/generic/hack-6.6/765-mxl-gpy-control-LED-reg-from-DT.patch using plaintext: 
patching file drivers/net/phy/mxl-gpy.c

Applying /scratch/union/target/linux/generic/hack-6.6/765-mxl-gpy-control-LED-reg-from-DT.patch.orig using plaintext: 
patching file drivers/net/phy/mxl-gpy.c
Hunk #1 FAILED at 10.
Hunk #2 FAILED at 38.
Hunk #3 FAILED at 61.
Hunk #4 succeeded at 235 (offset 7 lines).
Hunk #5 FAILED at 272.
4 out of 5 hunks FAILED -- saving rejects to file drivers/net/phy/mxl-gpy.c.rej
Patch failed!  Please fix /scratch/union/target/linux/generic/hack-6.6/765-mxl-gpy-control-LED-reg-from-DT.patch.orig!
make[4]: *** [Makefile:26: /scratch/union/build_dir/target-aarch64_cortex-a53_musl/linux-mediatek_filogic/linux-6.6.36/.prepared_92793dfbe3b467b74892ec768b3b4213] Error 1
make[4]: Leaving directory '/scratch/union/target/linux/mediatek'
make[3]: *** [Makefile:11: compile] Error 2
make[3]: Leaving directory '/scratch/union/target/linux'
time: target/linux/compile#5.90#4.65#7.95
    ERROR: target/linux failed to build.
make[2]: *** [target/Makefile:32: target/linux/compile] Error 1
make[2]: Leaving directory '/scratch/union'
make[1]: *** [target/Makefile:25: /scratch/union/staging_dir/target-aarch64_cortex-a53_musl/stamp/.target_compile] Error 2
make[1]: Leaving directory '/scratch/union'
make: *** [/scratch/union/include/toplevel.mk:248: world] Error 2
```
If I replace `target/linux/generic/hack-6.6/765-mxl-gpy-control-LED-reg-from-DT.patch` with @zxlhhyccc's version, it also fails to compile:
```
Applying /scratch/union/target/linux/generic/hack-6.6/765-mxl-gpy-control-LED-reg-from-DT.patch using plaintext: 
patching file drivers/net/phy/mxl-gpy.c

Applying /scratch/union/target/linux/generic/hack-6.6/765-mxl-gpy-control-LED-reg-from-DT.patch.orig using plaintext: 
patching file drivers/net/phy/mxl-gpy.c
Hunk #1 FAILED at 10.
Hunk #2 FAILED at 38.
Hunk #3 FAILED at 61.
Hunk #4 FAILED at 228.
Hunk #5 succeeded at 307 with fuzz 2 (offset 37 lines).
4 out of 5 hunks FAILED -- saving rejects to file drivers/net/phy/mxl-gpy.c.rej
Patch failed!  Please fix /scratch/union/target/linux/generic/hack-6.6/765-mxl-gpy-control-LED-reg-from-DT.patch.orig!
make[4]: *** [Makefile:26: /scratch/union/build_dir/target-aarch64_cortex-a53_musl/linux-mediatek_filogic/linux-6.6.36/.prepared_8726214b6526fee2cb5c59025ac3a2dc] Error 1
make[4]: Leaving directory '/scratch/union/target/linux/mediatek'
make[3]: *** [Makefile:11: compile] Error 2
make[3]: Leaving directory '/scratch/union/target/linux'
time: target/linux/compile#6.04#4.69#8.08
    ERROR: target/linux failed to build.
make[2]: *** [target/Makefile:32: target/linux/compile] Error 1
make[2]: Leaving directory '/scratch/union'
make[1]: *** [target/Makefile:25: /scratch/union/staging_dir/target-aarch64_cortex-a53_musl/stamp/.target_compile] Error 2
make[1]: Leaving directory '/scratch/union'
make: *** [/scratch/union/include/toplevel.mk:248: world] Error 2
```

graysky2 commented:
I deleted my previous comment due to after rebuilding my build system, then applying @DragonBluep's patch, I was able to successfully build `flogic/xiaomi_redmi-router-ax6000-ubootmod`.  I just cannot run test it for sometime.

zxlhhyccc commented:
To refresh the patch：
````
Run . .github/workflows/scripts/ci_helpers.sh
target/linux/mediatek/patches-6.6/732-net-phy-mxl-gpy-don-t-use-SGMII-AN-if-using-phylink.patch
Kernel patches for mediatek/filogic require refresh. (run 'make target/linux/refresh' and force push this pr)
You can also check the provided artifacts with the refreshed patch from this CI run.
Error: Process completed with exit code 1.
````

Noltari commented:
@graysky2 the upstreamed patch that is now removed from bmips is "010-v6.10-mips-bmips-BCM6358-make-sure-CBR-is-correctly-set.patch" and not "203-...".
Please adjust the commit description accordingly.
